enslig-forsorger/src/main/kotlin/no/nav/familie/kontrakter/ef/infotrygd/InfotrygdPerioder.kt

```diff
@@ -9,10 +10,42 @@ data class InfotrygdPerioderArenaRequest(val personIdenter: Set<String>,
                                          val fomDato: LocalDate? = null,
                                          val tomDato: LocalDate? = null)
 
-data class InfotrygdPerioderResponse(val perioder: List<InfotrygdPeriode>)
+data class InfotrygdPerioderArenaResponse(val perioder: List<InfotrygdArenaPeriode>)
+
+data class InfotrygdArenaPeriode(val personIdent: String,
+                                 val fomDato: LocalDate,
+                                 val tomDato: LocalDate,
+                                 val beløp: Float,
+                                 val opphørsdato: LocalDate? = null)
+
+data class InfotrygdPeriodeRequest(val personIdenter: Set<String>,
+                                   val stønadstyper: Set<StønadType>)
+
+data class InfotrygdPeriodeResponse(val perioder: List<InfotrygdPeriode>)
 
 data class InfotrygdPeriode(val personIdent: String,
-                            val fomDato: LocalDate,
-                            val tomDato: LocalDate,
-                            val beløp: Float,
-                            val opphørsdato: LocalDate? = null)
\ No newline at end of file
+                            val kode: InfotrygdEndringKode,
```

Comment:
Her kom jeg på att vi trenger å returnere stønadType også.
Si eks att man henter perioder for alle stønadstyper. Burde man då returnere alle periodene i en liste, eller burde an gruppere de per stønadstype? 
```
data class InfotrygdPeriodeResponse(val perioder: List<InfotrygdPeriode>)
data class InfotrygdPeriodeResponse(val overgangsstønad: List<InfotrygdPeriode>
                                                                 val barnetilstyn: List<InfotrygdPeriode>,
                                                                 val skolepenger: List<InfotrygdPeriode>)
data class InfotrygdPeriodeResponse(val perioder: Map<StønadType, List<InfotrygdPeriode>>)
```

Reply:
Jeg er ikke den beste til å svare på det, men tenker at de forskjellige stønadene behandles såpass forskjellig at det taler for å ha de i hver sin liste, slik som foreslått